tests/asyncapi-2.0/Message Object/valid.yaml

```diff
@@ -42,8 +42,8 @@ channels:
             email:
               type: string
               format: email
-        examples:
-          - one:
-              email: hello@foo.bar
-          - two:
-              email: bye@foo.bar
+          examples:
```

Comment:
but this change will make `examples` part of `payload` object, but should be `message` -> https://www.asyncapi.com/docs/specifications/2.0.0/#a-name-messageobject-a-message-object

Reply:
Hm, right. I guess I got confused by the fact that the Schema Object may have examples as well.  

@BlockLucas, is there any particular reason why you made this change?

Reply:
You are right, I got confused in the same way, will revert this file =)